test/integration/resultsPagePlace.js

```diff
@@ -57,9 +57,15 @@ describe('The place results page', () => {
           expect($(link).attr('href')).to.have.string('https://maps.google.com');
         });
 
+        const overviewLinks = $('.overview a');
+        overviewLinks.toArray().forEach((link) => {
+          expect($(link).attr('href')).to.have.string('https://www.nhs.uk/Services/pharmacies/Overview/DefaultView.aspx');
```

Comment:
Assuming the (to be added) unit tests are confirming the correct id has been appended to the base url, it would be good to extend the test to confirm the number of links i.e. 4 have been displayed on the page.
It wouldn't hurt to check the text of the link either.

Reply:
Done